met-api/src/met_api/models/submission.py

```diff
@@ -72,6 +83,49 @@ def update(cls, submission: SubmissionSchema) -> DefaultMethodResult:
         db.session.commit()
         return DefaultMethodResult(True, 'Submission Updated', submission_id)
 
+    @classmethod
+    def update_comment_status(cls, submission_id, status_id, reviewed_by, user_id):
+        """Update comment status."""
+        query = Submission.query.filter_by(id=submission_id)
+
+        if not query.first():
+            return DefaultMethodResult(False, 'Submission Not Found', submission_id)
```

Comment:
DefaultMethodResult is not used much, you could try None...

Reply:
raise error maybe, since it's not normal that submission is not found at this point